R/pkg/R/DataFrame.R

```diff
@@ -1375,8 +1375,8 @@ setMethod("saveDF",
 #' saveAsTable(df, "myfile")
 #' }
 setMethod("saveAsTable",
-          signature(df = "DataFrame", tableName = 'character', source = 'character',
-                    mode = 'character'),
+          signature(df = "DataFrame", tableName = "character", source = "character",
+                    mode = "character"),
           function(df, tableName, source = NULL, mode="append", ...){
```

Comment:
Maybe. But personally, I think fixing it would be better with another issue. Since we haven't discussed the validation rule for the spaces for parameters yet.